monkey/monkey_island/cc/ui/src/components/pages/LandingPage.tsx

```diff
@@ -74,4 +85,18 @@ function MonkeyInfo() {
   );
 }
 
+function ScenarioInfo() {
+    // TODO change links when added to documentation
+    return (
+      <>
+        <div className={'scenario-info'}>
+          Check the Infection Monkey documentation hub for more information
+          on <a href='https://www.guardicore.com/infectionmonkey/docs' rel="noopener noreferrer" target="_blank">
+           simulations
+          </a>.
```

Comment:
Should be "about scenarios" maybe? User is not choosing a simulation, he's choosing a scenario.

Reply:
IMO, we should use either "scenario" or "simulation". If we use "scenario", we should also reword the text on the buttons.